generators/lib/drupalProjectVersion.js

```diff
@@ -102,9 +102,20 @@ function init() {
 
   // Convert a semver version from '8.2.6' to a minor range '^8.2'.
   function toMinorRange(version) {
-    var regex = /^(\d+\.\d+)\.\d+/;
+    var regex = /^\d+\.\d+/;
     var range = version.match(regex);
-    return '^' + range[1];
+    if (range) {
+      return '^' + range[0];
+    }
+    else {
```

Comment:
Needs some code comments to explain the regex parsing that is going on here.  Not sure I understand what use-case the "else" clause is handling and why the previous simpler code was not working.